.vib/redmine/cypress/cypress/integration/redmine_spec.js

```diff
@@ -0,0 +1,81 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows admin to login/logout', () => {
+  cy.login();
+  cy.get('#flash_error').should('not.exist');
+  cy.contains('Sign out').click();
+  cy.contains('Sign in');
+});
+
+it('allows user to register', () => {
+  cy.visit('account/register');
+  cy.fixture('users').then((user) => {
+    cy.get('#user_login').type(`${user.newUser.userName}.${random}`);
+    cy.get('#user_password').type(`${user.newUser.password}.${random}`);
+    cy.get('#user_password_confirmation').type(
+      `${user.newUser.password}.${random}`
+    );
+    cy.get('#user_firstname').type(`${user.newUser.firstName}.${random}`);
+    cy.get('#user_lastname').type(`${user.newUser.lastName}.${random}`);
+    cy.get('#user_mail').type(`${user.newUser.email}.${random}@email.com`);
+  });
+  cy.get('[type="submit"]').click();
+  cy.contains('Your account was created');
+});
+
+it('allows admin to create a project and an issue with file uploaded', () => {
+  cy.login();
+  cy.visit('projects');
+  cy.contains('New project').click();
+  cy.fixture('projects').then((project) => {
+    cy.get('#project_name').type(`${project.newProject.name}.${random}`);
+    cy.get('#project_description').type(project.newProject.description);
+    cy.get('[name="commit"]').click();
+    cy.contains('Successful creation.');
+    cy.contains('Projects').click();
+    cy.contains('.project', `${project.newProject.name}.${random}`).click();
+  });
+  cy.contains('Issues').click();
+  cy.get('.icon-add').click();
+  cy.fixture('issues').then((issue) => {
+    cy.get('#issue_subject').type(`${issue.newIssue.subject}.${random}`);
+    cy.get('#issue_description').type(issue.newIssue.description);
+    cy.get('input[type="file"]').selectFile('cypress/fixtures/issues.json');
+  });
+  cy.get('[name="commit"]').click();
+  cy.contains('#flash_notice', 'created');
+  cy.contains('issues.json');
+});
+
+it('allows admin to check smtp configuration', () => {
+  cy.login();
+  cy.visit('/settings?tab=notifications');
+  cy.fixture('smtps').then((smtp) => {
+    cy.get('#settings_mail_from')
+      .clear()
+      .type(`${smtp.newSMTP.smtpEmailAddress}.${random}`);
+    cy.get('#settings_emails_header')
+      .scrollIntoView()
+      .clear()
+      .type(`${smtp.newSMTP.header}.${random}`);
+    cy.get('#settings_emails_footer')
+      .clear()
+      .type(`${smtp.newSMTP.footer}.${random}`);
+  });
+  cy.contains('[type="submit"]', 'Save').click({ force: true });
+  cy.contains('Successful update.');
+});
+
+it('allows admin to verify and change application configuration', () => {
+  cy.login();
+  cy.visit('/admin');
+  cy.contains('Settings').click();
+  cy.get('#settings_app_title').clear().type(`VMWare.${random}`);
+  cy.contains('input', 'Save').click();
+  cy.contains('#flash_notice', 'Successful update.');
+  cy.visit('/admin/info');
+  cy.contains('Mysql');
+  cy.contains('production');
+  cy.contains('Git');
```

Comment:
I'm not familiar with the asset but shouldn't we use `const` variables for these values?

Reply:
After your comment, I was indecisive about using `const` or fixtures.
Since we already have the `settings.json` file, I think it makes more sense to have this there. I've applied this change as well.

Reply:
Maybe a bit unrelated to this, is it necessary to check the page contains these 3 elements? I could be missing something as I'm not familiar with redmine, but we already checked the "Successful update" notice and these elements look like were already as they are before the update.

Reply:
The intention of the test is to **verify**(observe) the application configuration, as well as modify it. 
When we modify the configuration, we go to the info page which is under the admin tab. 
The tests are taken a look at this page:

![redmine](https://user-images.githubusercontent.com/47592680/167821691-2e6a385b-fae8-4828-a057-841952bc4456.png)

Since they are showing important components of this application (database, version control) I thought it is relevant to have them.